xdebug/view.py

```diff
@@ -100,6 +100,11 @@ def generate_context_output(context, indent=0):
     for variable in context.values():
         has_children = False
         property_text = ''
+        if variable['name']:
```

Comment:
If `variable['name']` isn't set, then `full_name` doesn't get created on new line 104.

If `variable` has children, then `full_name` will be referenced on new line 139, and an error will occur if it doesn't exist:

    UnboundLocalError: local variable 'full_name' referenced before assignment

So I think `full_name` needs to be created at the beginning of this `for` loop, eg: on line 102:

    full_name = ''